python/deploy

```diff
@@ -8,13 +8,31 @@ SOURCE_DIR=/var/lib/tsuru
 source ${SOURCE_DIR}/base/deploy
 source ${SOURCE_DIR}/base/rc/config
 
-APP_VENV=/home/application/.app_env
+APP_VENV=app_env
 
+#find apps python version
+PYTHON_VERSION=3.5.3  # default version if none pointed
```

Comment:
Maybe we could check if PYTHON_VERSION env is already set? So one can extend our public platform and change the default python version with easy. We do that for the ruby platform: https://github.com/tsuru/platforms/blob/master/ruby/deploy#L29